packages/frontend-shared/src/components/Select.vue

```diff
@@ -134,9 +136,11 @@
                     </span>
                   </slot>
                 </span>
-              </li>
+              </div>
             </ListboxOption>
-            <slot name="footer" />
+            <li role="option">
+              <slot name="footer" />
+            </li>
```

Comment:
I know you didn't add this, but looking at the code + screenshot, this is a super bizzare usage to place an info/link item in the select 😅

Reply:
It is!